### PR TITLE
fix: Push notification in iOS not working

### PR DESCRIPTION
**Pull request by mbfakourii**
## Pull Request

- Report security issues [confidentially](https://github.com/parse-community/Parse-SDK-Flutter/security/policy).
- Any contribution is under this [license](https://github.com/parse-community/Parse-SDK-Flutter/blob/master/LICENSE).
- Link this pull request to an [issue](https://github.com/parse-community/Parse-SDK-Flutter/issues?q=is%3Aissue).

## Issue
Changing `awesome_notifications` package to `flutter_local_notifications` in `ParseNotification`

Closes: #936

## Approach
n/a

## Tasks

- [ ] Add tests
- [ ] Add changes to documentation (guides, repository pages, code comments)

### CODE REVIEW

**Review by mtrezza — CHANGES_REQUESTED** [review]
Requires clarification of https://github.com/parse-community/Parse-SDK-Flutter/issues/936#issuecomment-1600600707

**Comment by mbfakourii** [comment]
@mtrezza 
Ready to merge?

**Comment by mtrezza** [comment]
Is this package change in any way a breaking change for developers, or is that only internally?

**Comment by mbfakourii** [comment]
> Is this package change in any way a breaking change for developers, or is that only internally?

No, it is not a breaking, only internally

**Comment by mtrezza** [comment]
Got it, then this would be a `refactor`, but that doesn't create a changelog entry and won't publish a release. Should we make a release and publish this as a `feat`?

File: packages/flutter/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## [5.1.1](https://github.com/parse-community/Parse-SDK-Flutter/compare/flutter-5.1.0...flutter-5.1.1) (2023-06-21)
+
+### Bug Fixes
+
+* Changing `awesome_notifications` package to `flutter_local_notifications` in `ParseNotification` ([#940](https://github.com/parse-community/Parse-SDK-Flutter/pull/940))
```
Comment (mtrezza) [inline packages/flutter/CHANGELOG.md:5]:
```suggestion
## [5.1.1](https://github.com/parse-community/Parse-SDK-Flutter/compare/flutter-5.1.0...flutter-5.1.1) (2023-06-28)

### Bug Fixes

* Push notifications in iOS not working; this changes the dependency in `ParseNotification` from `awesome_notifications` to `flutter_local_notifications` ([#940](https://github.com/parse-community/Parse-SDK-Flutter/pull/940))
```

**Review by mtrezza — CHANGES_REQUESTED** [review]

**Review by mtrezza — APPROVED** [review]

**Pull request merged**